src/js/components/DateInput/__tests__/DateInput-test.tsx

```diff
@@ -899,6 +899,51 @@ describe('DateInput', () => {
     expect(dateInputValue).toEqual('07/02/2020-07/07/2020');
   });
 
+  test('should be working with blur validation', async () => {
```

Comment:
Just a note, this test is working even without the fix in DateInput. The issue may not be reproducing in the jest testing environment and may be something we want to consider testing in an end to end environment. For now I am okay with leaving this test in and just adding a note in our end to end testing issue that we should replace this with an end to end test.